akashian/phase2-week1.md

```diff
@@ -0,0 +1,124 @@
+### **Week 1: Capture the Orders**
+
+The Akashian Challenge team will be creating orders continuously. This challenge asks users to explore how to operate as a provider by finding and bidding on leases. This can be accomplished by running the [provider daemon](https://github.com/ovrclk/akash/tree/master/provider), using `akashctl` and some `bash` scripting, or by building custom tooling. Contestants will be rewarded points during Week 1 for the following items:
+
+- Teams whose `Providers` have won the most number of `Orders`, then converted to `Leases`, put out by the Akash team
+- Teams whose `Providers` have placed the most number of `Bids` on `Orders`.
+- Bonus Category: Blog post detailing how winning provider bids works in the Akash system and some custom code to do so.
+- Bonus Category: Documentation or blog post detailing how the provider daemon works and how to operate it.
+
+### Creation of Provider
+
+In order to participate in the Akash network as a Provider of compute users will need to register with the network and point to where their services are running (`host`) as well as providing additional metadata (`attributes`). This metadata can be things like the region or area the computers are running in as well as additional information about what that Provider is actually providing. To start creating your provider on the testnet, create a `provider.yaml` file that contains at least the fields below:
+
+```yaml
+host: http://localhost:8080
+attributes:
+  - key: challenge
+    value: phase2
+  - key: team-name
+    value: your-team-name-here
+```
+
+Next you need to send this SDL file via a `akashctl tx provider create` command to the network. The key you sign and send this message with will be the key that you will need to use for all provider operations including capturing bids, so make sure this key has sufficent funds. The exact command looks like the following:
+
+> NOTE: Ensure that you have configured your node to talk to a valid RPC endpoint using `akashctl config node`
+
+```bash
+$ akashctl tx provider create "provider.yaml" --from mykey
+```
+
+You can check to ensure your provider has been created successfully by running the following query:
+
+```bash
+$ akashctl q provider get $(akashctl keys show mykey -a)
+```
+
+### Looking for Orders to Bid on
+
+During the competition the Akash team will be submitting `Deployments` periodically. Those deployment requests will create `Orders` on the chain. They will be originating from the following addresses:
+
+```
+TODO: add list of addresses where orders will be originating
```

Comment:
Note: We need to add a list of the addresses we will be using during the testnet here